app/views/buyers/accounts/_account_details.html.slim

```diff
@@ -8,9 +8,10 @@ table.list
         => link_to 'Impersonate',
                   admin_buyers_account_impersonation_path(account),
                   class: 'action bolt', method: 'post', target: '_blank'
-      => link_to 'Send message',
-                new_provider_admin_messages_outbox_path(to: account),
-                class: 'action message fancybox'
+      - unless account.suspended_or_scheduled_for_deletion?
+        => link_to 'Send message',
+                  new_provider_admin_messages_outbox_path(to: account),
+                  class: 'action message fancybox'
```

Comment:
This button:

![image](https://github.com/user-attachments/assets/98213faf-857c-4afe-ba1f-e0a71fdc38ab)

Reply:
Yes, the code totally allows it, the mdoel is `Account` for both buyers and providers. Also, this screenshot is from a buyer and you can see there is a "Suspend" button there. When the buyer is scheduled for deletion, you can see a "Resume" button here. I don't know if this is actually used but there's nothing that prevents suspending a buyer.

Reply:
right, we are not scheduling for deletion individual buyers but we do suspend them. Makes sense.